metricshub-engine/src/main/java/org/sentrysoftware/metricshub/engine/strategy/source/SourceTable.java

```diff
@@ -106,7 +106,7 @@ public static String tableToCsv(
 	public static List<List<String>> csvToTable(final String csvTable, final String separator) {
 		if (csvTable != null) {
 			return Stream
-				.of(csvTable.split("\n"))
+				.of(csvTable.split("\n|\r"))
```

Comment:
This lead to an empty line.

Reply:
Regex should be:

```java
csvTable.split("\\r?\\n|\\r");
```

Reply:
Actually this change has been removed as it was unnecessary since we forced the end of line character of Jawk Sources to `\n`.